toolchain-powerpc.cmake

```
@@ -2,8 +2,8 @@ set(CMAKE_SYSTEM_NAME "Generic")
 set(CMAKE_SYSTEM_VERSION 1)
 set(CMAKE_SYSTEM_PROCESSOR powerpc-eabi)
 
-set(DEVKITPRO $ENV{DEVKITPRO})
-set(DEVKITPPC $ENV{DEVKITPPC})
+file(TO_CMAKE_PATH $ENV{DEVKITPRO} DEVKITPRO)
```

```suggestion
file(TO_CMAKE_PATH "$ENV{DEVKITPRO}" DEVKITPRO)
```

From the CMake docs:

> Always use double quotes around the <path> to be sure it is treated as a single argument to this command.